_config.yml

```diff
@@ -26,8 +28,8 @@ permalink:        pretty
 
 # Setup
 title:            The OpenTracing Project
-tagline:          'Common APIs for distributed tracing and context propagation'
-description:      'Common APIs for distributed tracing and context propagation'
+tagline:          'Safe, general APIs for distributed tracing and context propagation'
+description:      'Safe, general APIs for distributed tracing and context propagation'
```

Comment:
yeah, "safe" is the wrong word... I changed this and also added a little more detail to the "APIs" page.